get_started_with_gx_cloud/workshop.md

```diff
@@ -147,6 +150,58 @@ We've already created this expectation, so go ahead and cancel creating a nw exp
 
 <img src="img/new_expectation_with_metrics.png" alt="Create a new Expectation using Data Asset Metrics" style="width:500px;"/><br>
 
+## Custom SQL Expectations
+You can create custom SQL Expectations in GX Cloud. These will fail validation if the SQL query returns one or more rows, so you can perform any query against the data that you wish.
+
+Let's create a new expectation using the custom SQL Expectation form. Since we are using the demo data set and we don't have access to query the data directly, we'll use the example query that is filled in by default.
+
+> **Create Custom SQL Expectation**
+> 1. On the Expectations tab, click **New Expectation**
+> 1. Click **Create custom Expectation** at the bottom of the panel
+> 1. Enter the description, "**Manhattan Passenger Count**"
+> 1. Review the SQL query, it should appear as below.
+> 1. Click **Save**.
+
+Note that in this query, we're selecting all rows for the "`Manhattan`" `pickup_borough` with a `passenger_count` of `4`.
+
+<img src="img/new_sql_expectation.png" style="width:500px;"/><br>
+
+Now that the SQL Expectation is created, run the Validation again. Go to the Validations tab again and see that the Expectation failed. As we saw from the earlier exercise, this is because we will have rides that used taxis with up to 6 seats. When creating this Expectation, we may assume that rides to Manhattan were in smaller cabs because the streets are more crowded, but upon further investigation we know that isn't the case.
+
+<img src="img/failed_validation_sql.png" style="width:500px;"/><br>
+
+Edit the Expectation, and in the SQL code box, change `passenger_count > 4` to `passenger_count > 6`. Rerun the Validation again and view the results.
```

Comment:
I think it would be better if we could use a custom sql expectation that is distinctly different than the expectations we've already used in the workshop. Maybe do something unusual, like checking that a specific vendor never charges more than a certain amount for the airport fee?

Reply:
Done! See latest commit.